### DISCUSSION TITLE
codespaces stopped working

### DISCUSSION THREAD

**Discussion by mspat227** [community/community#45714] [Codespaces] [answered]
### Select Topic Area

Question

### Body

![Screenshot (8)](https://user-images.githubusercontent.com/108686479/215333545-94509f78-c2f8-495a-908a-4729845ee975.png)

My code spaces stopped working suddenly. when i logged in with GitHub account above screen appears and nothing happens after. i have been using it for CS50 for almost 9 months and didn't had any problem till today. anyone with a solution?

**Comment by jge162** [comment, marked as answer]
Maybe your computer is out of resources, look at the amount of tabs you have open. clear the cache, restart your computer to stop all running processes and try again with a new web browsers.

**Reply by mspat227** [reply]
Great. Thank you so much. I have cleared the cache in browser and then in PC too. now its working well.

**Comment by brunoalksouza** [comment]
maybe cold be new web browsers
